docs/engines.md

```diff
@@ -159,5 +159,18 @@ config.middleware.use(
   urls: ["/my-engine-packs"], root: "my_engine/public"
 )
 ```
+or if you prefer to keep your engine-related configuration within the engine itself
+
+```ruby
+# my-engine-root/lib/my-engine/engine.rb
+module MyEngine
+  class Engine < ::Rails:Engine
+    config.app_middleware.use(
+      "Rack::Statuc",
```

Comment:
Typo here ^^ should be `Rack::Static` no?

Reply:
indeed... I've fixed it, thanks for catching it